src/app/dataValidator.jsx

```diff
@@ -0,0 +1,101 @@
+const yaml = require('yaml-js'); // eslint-disable-line import/no-extraneous-dependencies
+const fs = require('fs'); // eslint-disable-line import/no-extraneous-dependencies
+
+const yamlSchema = fs.readFileSync('././data/schema.yaml', 'utf8');
+
+const { components } = yaml.load(yamlSchema);
+const { schemas } = components;
+
+const log = message => {
+  if (process.env.NODE_ENV !== 'test' && typeof jest === 'undefined') {
```

Comment:
Can you add a comment here, explaining why you've added these two conditions?

Reply:
We will replace these with `global.console.log = jest.fn();` in the jest test files.